bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Device.java

```diff
@@ -645,7 +645,6 @@ public Color getSystemColor (int id) {
 		case SWT.COLOR_DARK_GREEN:			pixel = 0x00008000;  break;
 		case SWT.COLOR_YELLOW:				pixel = 0x0000FFFF;  break;
 		case SWT.COLOR_DARK_YELLOW:			pixel = 0x00008080;  break;
-		case SWT.COLOR_LIGHT_BLUE:			pixel = 0x00FF9933;  break;
```

Comment:
If you introduce such new API you shoud do it for Cocoa and GTK too.

Reply:
We are now sticking to the default color that is already present i.e. COLOR_DARK_GRAY